docs/quickstart.rst

```diff
@@ -26,18 +22,20 @@ For example, to load a `SDSS MaNGA IFU data cube <https://data.sdss.org/sas/dr14
 In a Jupyter Notebook
 ---------------------
 
-``jdaviz`` provides a directory of sample notebooks to test the application, located in the ``notebooks`` sub-directory
-of the git repository.  ``Example.ipynb`` is provided as an example that loads a SDSS MaNGA IFU data cube with the
-``Cubeviz`` configuration.  To run the provided example, start the jupyter kernel with the notebook path::
+The power of ``jdaviz`` is that it can integrated into your Juppyter notebook workflow::
 
-    jupyter notebook /path/to/jdaviz/notebooks/Example.ipynb
+    from jdaviz import Specviz
 
-or simply start a new Jupyter notebook and run the following in a cell::
+    app = Specviz()
+    app
+To learn more about the various ``jdaviz`` application configurations and loading data, see the
+`specviz <https://jdaviz.readthedocs.io/en/latest/specviz/import_data.html>`_, `cubeviz <https://jdaviz.readthedocs.io/en/latest/cubeviz/import_data.html>`_, `mosviz <https://jdaviz.readthedocs.io/en/latest/mosviz/import_data.html>`_, or `imviz <https://jdaviz.readthedocs.io/en/latest/imviz/import_data.html>`_ tools.
```

Comment:
Should use `:ref:` instead of hardcoding the URL. Not all pages mentioned is set up for `:ref:`, so that needs to be fixed too as part of this work.